numpy/lib/_function_base_impl.pyi

```diff
@@ -15,8 +15,9 @@ from typing import (
 )
 from typing_extensions import deprecated
 
+import numpy as np
 from numpy import (
-    vectorize as vectorize,
```

Comment:
Remind me of why `vectorize as vectorize` makes sense. I've forgotten.

Reply:
In this case it doesn't, but if there's no `__all__`, an `import spatula from spatula` will implicitly export `spatula` as a public member of that module. It's almost always better to use `__all__`, but mypy's `stubgen` tool used to (since last month) generate stubs with such implicit exports.